packages/gatsby-source-contentful/src/__tests__/download-with-retry.js

```diff
@@ -0,0 +1,83 @@
+import nock from "nock"
```

Comment:
This should set jest-environment to node for this test
https://jestjs.io/docs/en/configuration#testenvironment-string

```suggestion
/**
 * @jest-environment node
 */
import nock from "nock"
```

Reply:
i did not know! let me try :)

Reply:
#done